api/src/Controller/Department/PutDepartment.php

```diff
@@ -0,0 +1,120 @@
+<?php declare(strict_types=1);
+
+/**
+ * @OA\Put(
+ *     tags={"departments"},
+ *     path="/department/{id}",
+ *     summary="Update existing department",
+  *    @OA\Parameter(
+ *         description="ID of department being updated",
+ *         in="path",
+ *         name="id",
+ *         required=true,
+ *         @OA\Schema(type="integer")
+ *     ),
+ *     @OA\RequestBody(
+ *         @OA\MediaType(
+ *             mediaType="multipart/form-data",
+ *             @OA\Schema(
+ *                 @OA\Property(
+ *                     property="Name",
+ *                     description="The name of the department",
+ *                     nullable=true,
+ *                     type="string"
+ *                 ),
+ *                 @OA\Property(
+ *                     property="ParentID",
+ *                     description="The ID for the parent department",
+ *                     nullable=true,
+ *                     type="integer"
+ *                 ),
+ *                 @OA\Property(
+ *                     property="FallbackID",
+ *                     description="The ID for the fallback department",
+ *                     nullable=true,
+ *                     type="integer"
+ *                 )
+ *             )
+ *         )
+ *     ),
+ *     @OA\Response(
+ *         response=200,
+ *         description="OK",
+ *         @OA\JsonContent(
+ *             ref="#/components/schemas/department"
+ *         )
+ *     ),
+ *     @OA\Response(
+ *         response=401,
+ *         ref="#/components/responses/401"
+ *     ),
+ *     @OA\Response(
+ *         response=400,
+ *         ref="#/components/responses/400"
+ *     ),
+ *     security={{"ciab_auth":{}}}
+ * )
+ */
+
+namespace App\Controller\Department;
+
+use App\Error\InvalidParameterException;
+use Slim\Container;
+use Slim\Http\Request;
+use Slim\Http\Response;
+
+use App\Service\DepartmentService;
+
+class PutDepartment extends BaseDepartment
+{
+
+    /**
+     * @var DepartmentService
+     */
+    protected $departmentService;
+
+
+    public function __construct(Container $container)
+    {
+        parent::__construct($container);
+        $this->departmentService = $container->get("DepartmentService");
+        $this->includes = null;
+
+    }
+
+
+    public function buildResource(Request $request, Response $response, $params): array
+    {
+        $departmentId = $params["id"];
+        $body = $this->checkRequiredBody($request, []);
```

Comment:
I do not this this is required with an empty set.

Reply:
OK, will remove this.